apps/pie-storybook/stories/pie-modal.stories.ts

```diff
@@ -31,7 +31,7 @@ const defaultArgs: ModalProps = {
     isLoading: false,
     size: 'medium',
     position: 'center',
-    slot: '<p>Body copy</p>',
+    slot: 'Body copy',
```

Comment:
I thought we specifically wanted to use an html tag here because its how real users will add slots to our component? That's why we added that html sanitisation helper. How come we are removing this?

Reply:
yes, but this is just a simple example and @RLumleyJET would like to align it to the design's example for consistency.

Reply:
as discussed, that's fine but we can still use HTML here - I would suggest a `<span>` :)